src/trio/_tests/test_testing_raisesgroup.py

```diff
@@ -29,7 +29,12 @@ def test_raises_group() -> None:
     with RaisesGroup(ValueError):
         raise ExceptionGroup("foo", (ValueError(),))
 
-    with RaisesGroup(SyntaxError):
+    with pytest.raises(
+        AssertionError,
+        match=wrap_escape(
+            "Raised exception group did not match: SyntaxError() is not of type 'ValueError'",
```

Comment:
I'm somewhat skeptical of the parenthesized-with here because we're testing the context manager, rather than the innermost body being what the focus is on, but testing it just for the sake of exciting "new" feature ^^